### PR TITLE
v8 Create + Broadcast 

### PR DESCRIPTION
**Pull request by micahriggan**
I had a successful run - through with utxos -> create tx -> [sign tx in bitcoin-core] -> broadcast tx

```
./packages/bitcore-client/bin/wallet-utxos --path ~/regtest.wallet.dat/

{"txid":"0b62bcdb754947be528ddc3b358549d2037da153cabbb26a61ef98e4f11147ab","vout":0,"address":"n2NXZ78CYEBzaoHMHJhy7ji7NPdGJZtymH","script":"76a914e4c2a95527190462044fa82f9840c180cd14e72088ac","value":2500000000}

```

```
./packages/bitcore-client/bin/wallet-tx --path ~/regtest.wallet.dat/ --utxos '[{"txid":"0b62bcdb754947be528ddc3b358549d2037da153cabbb26a61ef98e4f11147ab","vout":0,"address":"n2NXZ78CYEBzaoHMHJhy7ji7NPdGJZtymH","script":"76a914e4c2a95527190462044fa82f9840c180cd14e72088ac","value":2500000000}]' --amount 2500000000 --addresses mz2KX9o4DdgqgQ8h7PPtdoNcLhgAqY6Q7S --change mhNYEc8ERzdNu6ahPdKnXP9wLZCAZkX9L2

:: DEBUG address and value
mz2KX9o4DdgqgQ8h7PPtdoNcLhgAqY6Q7S 2500000000

:: Raw TX
0100000001ab4711f1e498ef616ab2bbca53a17d03d24985353bdc8d52be474975dbbc620b0000000000ffffffff0100f90295000000001976a914cb000ba197e1851cc93c90af7a484b156b46f26888ac00000000
```

```
bitcoin-core signrawtransaction 0100000001ab4711f1e498ef616ab2bbca53a17d03d24985353bdc8d52be474975dbbc620b0000000000ffffffff0100f90295000000001976a914cb000ba197e1851cc93c90af7a484b156b46f26888ac00000000

{
  "hex": "0100000001ab4711f1e498ef616ab2bbca53a17d03d24985353bdc8d52be474975dbbc620b000000006b483045022100e9816b838cd6b8c173b2f2df6afa8645939bda3dcb5483e148feec75ce4c510302203292998df12aa1aec55fa7bfd9e649264159ee4b25505787f0b1ea0796de5e7b0121030ee38e5fea855fced8abf0ba5aa5e44a9793be7a81d42ab7f92d1e8d3082607bffffffff0100f90295000000001976a914cb000ba197e1851cc93c90af7a484b156b46f26888ac00000000",
  "complete": true
}
```

```
 ./packages/bitcore-client/bin/wallet-broadcast --path ~/regtest.wallet.dat/ --tx 0100000001ab4711f1e498ef616ab2bbca53a17d03d24985353bdc8d52be474975dbbc620b000000006b483045022100e9816b838cd6b8c173b2f2df6afa8645939bda3dcb5483e148feec75ce4c510302203292998df12aa1aec55fa7bfd9e649264159ee4b25505787f0b1ea0796de5e7b0121030ee38e5fea855fced8abf0ba5aa5e44a9793be7a81d42ab7f92d1e8d3082607bffffffff0100f90295000000001976a914cb000ba197e1851cc93c90af7a484b156b46f26888ac00000000
Transaction broadcasted
```

### CODE REVIEW

File: packages/bitcore-client/bin/wallet-import [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
 
 program
   .version(require('../package.json').version)
   .option('--path <path>', 'REQUIRED - Where wallet is stored')
+  .option('--load [load]', 'OPTIONAL - A jsonl file with {"address": "mXy1234"} in it')
   .parse(process.argv);
 
 const main = async () => {
-  const {path} = program;
+  const {path, load} = program;
```
Comment (micahriggan) [inline packages/bitcore-client/bin/wallet-import:14]: May want to change this to --file

File: packages/bitcore-client/lib/wallet.js [hunk 2]
```diff
@@ -71,11 +72,23 @@ class Wallet {
...
 
-  static async broadcast(params){
+  async newTx(params){
     const payload = {
       network: this.network,
       chain: this.chain,
-      rawTx: params.rawTx
+      addresses: params.addresses,
+      amount: params.amount,
+      utxos: params.utxos || await this.getUtxos(params),
```
Comment (micahriggan) [inline packages/bitcore-client/lib/wallet.js:81]: need to have a utxo selection algorithm to pick the best ones.

File: packages/bitcore-lib/lib/transaction/unspentoutput.js [hunk 3]
```diff
@@ -44,9 +44,9 @@ function UnspentOutput(data) {
   $.checkArgument(!_.isUndefined(data.scriptPubKey) || !_.isUndefined(data.script),
                   'Must provide the scriptPubKey for that output!');
   var script = new Script(data.scriptPubKey || data.script);
-  $.checkArgument(!_.isUndefined(data.amount) || !_.isUndefined(data.satoshis),
-                  'Must provide an amount for the output');
   var amount = !_.isUndefined(data.amount) ? new Unit.fromBTC(data.amount).toSatoshis() : data.satoshis;
+  amount = amount || data.value;
```
Comment (micahriggan) [inline packages/bitcore-lib/lib/transaction/unspentoutput.js:48]: our new utxo uses value. 
value === satoshis or any base unit of currency

File: packages/bitcore-node/lib/routes/wallet.js [hunk 4]
```diff
@@ -92,89 +84,6 @@ router.post('/:pubKey', authenticate, async (req, res) => {
   }
 });
 
-function ListTransactionsStream(walletId) {
```
Comment (micahriggan) [inline packages/bitcore-node/lib/routes/wallet.js:95]: This was dead code

**Review by micahriggan — COMMENTED** [review]
Should be ready for review

**Review by crypto0sy80 — APPROVED** [review]

**Pull request closed without merging**